Documentation/etcd-mixin/mixin.libsonnet

```diff
@@ -8,6 +8,26 @@
       {
         name: 'etcd',
         rules: [
+          {
+            alert: 'etcdMembersDown',
+            expr: |||
+              max by (job) (
+                sum by (job) (up{%(etcd_selector)s} == bool 0)
+              or
+                count by (job,endpoint) (
+                  sum by (job,endpoint,To) (rate(etcd_network_peer_sent_failures_total{%(etcd_selector)s}[3m])) > 0.01
+                )
+              )
+              > 0
+            ||| % $._config,
+            'for': '3m',
+            labels: {
+              severity: 'critical',
+            },
+            annotations: {
+              message: 'etcd cluster "{{ $labels.job }}": members are down ({{ $value }}).',
```

Comment:
Small nit, Is the etcd member down or unavailable in this case? For example in a network partition etcd could be running fine yet reporting as down. We are using a network-based metric ` etcd_network_peer_sent_failures_total` to report health according to the alert. Is that what we want?

`message: 'etcd cluster "{{ $labels.job }}": members are unavailable ({{ $value }}).',`?

Reply:
To be more clear we are using a metric from the perspective of a peers connection to another peer.

A <--> B is OK
C <--> A is OK
B <--> A is OK
[..]
A <-/-> C Not OK # alert

Reply:
I 100% agree we would want to know about the issue.

Reply:
Yeah, terminology is tough.  I think in practice down is the most common reason (with a small chance of unavailable or unreachable).  Since 95% of the failure modes are "down", I think I'd be ok emphasizing that part, and if you do have a partial partition (even one way) from the purposes of the quorum it might still be down.  I tried to put on my "what am I telling someone who knows nothing about network partitions" hat - for them, it could be many things that they can't really diagnose, but the majority are the simplest (pod evicted / disappeared from service discovery, thus "down").